### DISCUSSION TITLE
Allow Date field type to be matched on date without time

### DISCUSSION THREAD

**Discussion by timeverts** [engram-design/FeedMe#876] [Ideas]
A feature request.

Suppose our entries have a Craft Date field type which is configured to input a time component as well. When I import entries, I want to use that field in the 'Set a unique identifier for existing elements' but I would like to be able to indicate to Feed Me that I only want to match on the date portion of that field's value (ignoring the time), and that's what it would do during the import.

**Comment by engram-design** [comment]
I feel this might be needlessly complicated as a case to handle. Are you not able to create a custom plain text field that stores just the date value, and matching against that? I just don't want to confuse people giving them 3 options per date field on matching (date+time, date, time).
